src/main/java/net/minecraftforge/network/ForgeNetwork.java

```diff
@@ -0,0 +1,283 @@
+package net.minecraftforge.network;
+
+import java.util.Set;
+import java.util.stream.Stream;
+
+import io.netty.buffer.Unpooled;
+import net.minecraft.network.Connection;
+import net.minecraft.network.FriendlyByteBuf;
+import net.minecraft.network.protocol.Packet;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.server.level.ServerChunkCache;
+import net.minecraft.server.level.ServerLevel;
+import net.minecraft.server.level.ServerPlayer;
+import net.minecraft.server.network.ServerPlayerConnection;
+import net.minecraft.world.entity.Entity;
+import net.minecraft.world.entity.EquipmentSlot;
+import net.minecraft.world.entity.LivingEntity;
+import net.minecraft.world.inventory.AbstractContainerMenu;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.level.ChunkPos;
+import net.minecraft.world.level.block.entity.BlockEntity;
+import net.minecraftforge.common.ForgeMod;
+import net.minecraftforge.fmllegacy.network.NetworkDirection;
+import net.minecraftforge.fmllegacy.network.NetworkRegistry;
+import net.minecraftforge.fmllegacy.network.simple.SimpleChannel;
+import net.minecraftforge.network.message.BlockEntityCapabilitiesMessage;
+import net.minecraftforge.network.message.EntityCapabilitiesMessage;
+import net.minecraftforge.network.message.EquipmentSlotCapabilitiesMessage;
+import net.minecraftforge.network.message.SlotCapabilitiesMessage;
+
+public class ForgeNetwork {
```

Comment:
As I said for the 1.16 PR this is the main concern for this PR, do we want to go with forge own network channel. It's aggressive but it also offers a performance gain during capabilities sync.

I really want to see a discussion around it with forge team/triage team.